daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/svalue/Ordering.scala

```diff
@@ -223,15 +224,15 @@ object Ordering extends scala.math.Ordering[SValue] {
             case SAny(t2, v2) =>
               compareType(t1, t2) -> ImmArray((v1, v2))
           }
-          case SContractId(_) => {
-            case SContractId(_) =>
-              throw new IllegalAccessException("Try to compare relative contract ids")
+          case SContractId(RelativeContractId(_)) => {
```

Comment:
When are we planning to switch the default in sandbox to use absolute contract ids everywhere? I expect that having maps indexed by contract ids will be one of the major usecases for genmaps.

Reply:
I am not sure, I will change the default in the sandbox ever.
In kvutils (and then in next gen sandbox) it is already the only possible mode. 
DAML-LF 1.9 will require the new contract ids scheme to be enable.